src/gpu/nvidia/cudnn_batch_normalization_executor.hpp

```diff
@@ -198,27 +205,24 @@ struct bnorm_exec_fwd_inf_t : public bnorm_exec_base_t {
             const override {
         nvidia::sycl_cuda_stream_t *cuda_stream
                 = utils::downcast<nvidia::sycl_cuda_stream_t *>(ctx.stream());
-
-        auto wkspace_storage = bnorm_impl->is_training()
-                ? ctx.output(DNNL_ARG_WORKSPACE)->memory_storage()
-                : &memory_storage_t::empty_storage();
-
         auto n_channels = bnorm_impl->C();
-        ::sycl::buffer<float> scaleshift_buff(n_channels * 2);
+        ::sycl::buffer<float> scale_bias_buff(n_channels * 2);
+
         return cuda_stream->interop_task([&](::sycl::handler &cgh) {
-            auto src_acc = CTX_IN_ACCESSOR(DNNL_ARG_SRC);
-            auto dst_acc = CTX_OUT_ACCESSOR(DNNL_ARG_DST);
-            auto scale_acc
-                    = scaleshift_buff.get_access<::sycl::access::mode::write>(
-                            cgh, n_channels, 0);
-            auto bias_acc
-                    = scaleshift_buff.get_access<::sycl::access::mode::write>(
-                            cgh, n_channels, n_channels);
+            auto arg_src = CTX_IN_SYCL_MEMORY(DNNL_ARG_SRC);
+            auto arg_dst = CTX_OUT_SYCL_MEMORY(DNNL_ARG_DST);
+            auto arg_scale_bias = CTX_INTERNAL_SYCL_MEMORY(scale_bias_buff);
+            auto arg_wkspace = bnorm_impl->is_training()
+                    ? CTX_OUT_SYCL_MEMORY(DNNL_ARG_WORKSPACE)
+                    : CTX_NULL_SYCL_MEMORY();
+
             bool init_ss = true, init_mean_var = false;
 
-            interop_task_fwd(bnorm_impl, engine, cgh, cuda_stream, src_acc,
-                    dst_acc, nullptr, nullptr, scale_acc, bias_acc,
-                    wkspace_storage, init_ss, init_mean_var);
+            auto null_mem_obj = CTX_NULL_SYCL_MEMORY();
```

Comment:
I'd suggest to explicitly create an empty arg with a particular access mode. The way how `sycl_memory_arg_t` is defined now doesn't allow to have a single kind of empty arg.

Reply:
Change made. Thanks